x-pack/plugins/security_solution/public/management/services/exceptions_list/exceptions_list_api_client.ts

```diff
@@ -43,9 +43,11 @@ export class ExceptionsListApiClient {
       T extends CreateExceptionListItemSchema | UpdateExceptionListItemSchema
     >(
       item: T
-    ) => T
+    ) => T,
+    public readonly version?: string
```

Comment:
This will allow us to use different api versions for any endpoint exception type, if not defined at specific client instance (Trusted Apps, Event Filters, etc), it defaults to the default date.